packages/usa-in-page-navigation/src/test/template.html

```diff
@@ -0,0 +1,31 @@
+<div class="usa-in-page-nav-container">
+  <main id="main-content" class="main-content">
+    <h1>In page navigation test heading</h1>
+
+    <h2><a id="section_0" class="usa-anchor-tag"></a>Section 1</h2>
```

Comment:
Doesn't affect unit tests, but the component preview uses `usa-anchor` instead of `usa-anchor-tag`.

Reply:
@mitchmoccia let's make a note of this for a follow-up fix